stable/database/templates/statefulset.yaml

```diff
@@ -68,15 +68,9 @@ spec:
       - name: init-disk
         image: {{ template "init.image" . }}
         imagePullPolicy: {{ default "" .Values.busybox.image.pullPolicy | quote }}
-        command:
-          - 'chmod'
-          - '-R'
-          - '770'
-          - '/var/opt/nuodb/archive'
-          - '/var/log/nuodb'
-  {{- if eq (include "defaultfalse" .Values.database.sm.noHotCopy.journalPath.enabled) "true"}}
-          - '/var/opt/nuodb/journal'
-  {{- end }}
+        command: ['sh', '-c', '
```

Comment:
Same change needs to be applied on the hotcopy init-disk container below

Reply:
+1

Reply:
Good catch.